contracts/src/v0.8/vrf/VRFV2PlusSubscriptionManager.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
```

Comment:
The biggest value-add here is a "one-click" migration to a new VRF coordinator. Since this contract will be the subscription admin, you have to wrap all the operations on a subscription so that the actual admin (probably an EOA or a Gnosis safe if multisig) can perform admin operations like adding consumers and such. 

Re: stuff that may not exist in a future coordinator, the V2+ contracts will always have the same exact billing API/ABI (something that we want to keep stable) since all migrations will be only for implementation level fixes.